src/core_plugins/kibana/public/dashboard/listing/get_dashboard_columns.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import { getTitleColumn } from 'ui/saved_object_table/get_title_column';
+import { getCheckBoxColumn } from 'ui/saved_object_table/get_checkbox_column';
+
+export function getDashboardColumns(itemTableState) {
+  return [
+    getCheckBoxColumn(itemTableState),
+    getTitleColumn(itemTableState)
```

Comment:
(Yellow flag) This is a pretty opaque interface to me, because I can't infer what's going to be returned by either the name of the function or the argument